package.json

```diff
@@ -28,6 +28,7 @@
     "crypto-browserify": "^3.12.0",
     "dayjs": "^1.8.34",
     "ethers": "^5.5.1",
+    "fork-bn.js": "^4.11.8",
```

Comment:
What does this do?

Reply:
Bumped into this when I was running the bundle analyzer where it showed multiple chunks of `bn.js` library (an internal dependency of some other modules) in the bundle.

When searched for different ways to remove the duplicates of the `bn.js`, I bumped into [this issue](https://github.com/ChainSafe/web3.js/issues/1178) where they were also facing the same problem as me because of duplicate chunks of `bn.js`.

And in [this comment](https://github.com/ChainSafe/web3.js/issues/1178#issuecomment-464617646) on the same issue, there was a suggestion to install this `fork-bn.js` and alias it in the webpack config.

That helped me in reducing the bundlesize by 46kb.

Reply:
Cool! Never heard of this, but nice catch.

Reply:
Found the sourcecode of the [`fork-bn.js`](https://github.com/BigMurry/bn.js).

The only difference to the main library is that he changed the name of the package to `fork-bn.js` and re-published it. When you see the [diff](https://github.com/indutny/bn.js/compare/master...BigMurry:master) to the main repo, it is nothing but change in the name of the package.

The reason for deduction: since we have added this package as a direct dependency, webpack aliases all the `bn.js` chunks to `fork-bn.js` (magic of webpack aliases 😛)